pkg/kubelet/prober/prober_test.go

```diff
@@ -73,13 +92,23 @@ func TestGetURLParts(t *testing.T) {
 				},
 			},
 		}
-		p, err := extractPort(test.probe.Port, container)
+
+		scheme := test.probe.Scheme
+		if scheme == "" {
```

Comment:
this defaulting should go in pkg/api/v1/defaults.go and pkg/api/v1beta3/defaults.go